src/dialects/abstract/query-interface.js

```diff
@@ -448,7 +448,13 @@ export class QueryInterface {
     options = options || {};
     attribute = this.sequelize.normalizeAttribute(attribute);
 
-    return await this.sequelize.queryRaw(this.queryGenerator.addColumnQuery(table, key, attribute), options);
+    const addColumnQueryOptions = {
+      ifNotExists: options.ifNotExists,
+    };
+
+    delete options.ifNotExists;
```

Comment:
Preventing ifNotExists from going to queryRaw is good, but we must avoid mutating our inputs

Prefer this instead:

```
const { ifNotExists, ...queryRawOptions } = options;
```